src/components/SensorDiscoveryModal/index.tsx

```diff
@@ -33,7 +33,7 @@ export function SensorDiscoveryModal(props: Props) {
 
   const [scanning, setScanning] = useState(false);
   const [discovered, setDiscovered] = useState([]);
-  const containerStyle = {backgroundColor: 'white', padding: 20};
+  const modalContainerStyle = {backgroundColor: 'black', padding: 20};
```

Comment:
If you remove background color completely would it not pull a background color from the theme?

Reply:
I tried that but the default was transparent, so you just see the under layer sentence about adding sensors.